R/FeatSelControl.R

```diff
@@ -109,12 +111,14 @@
 NULL
 
 makeFeatSelControl = function(same.resampling.instance, impute.val = NULL, maxit, max.features,
-  tune.threshold = FALSE, tune.threshold.args = list(), log.fun = NULL, ..., cl) {
+  tune.threshold = FALSE, tune.threshold.args = list(), log.fun = "default", ..., cl) {
```

Comment:
Why the detour with the string. We could use the function names directly and put the defaults in the signature. But then we have to export these functions. :thinking: